js/tabs.js

```diff
@@ -376,7 +379,12 @@ Tabs.prototype.saveAs = function(opt_tab, opt_close) {
   }
   this.chooseEntry(
       {'type': 'saveFile', 'suggestedName': suggestedName},
-      this.onSaveAsFileOpen_.bind(this, opt_tab, opt_close || false));
+      function(entry) {
```

Comment:
I'm sure I've asked this before; can we use arrow functions here? So instead of

function(entry) {

}.bind(this);

just

(entry) => {
}

Reply:
Sorry I don't recall any previous conversations about arrow functions.

I can't add any ES6 features without changing the compilation language to es6 for closure (otherwise errors), and I can't do that without updating the CodeMirror library (cause compiling with es6 turns a CodeMirror warning into an error), and that's a big job since we've got a lot of outdated links to CodeMirror files, as well as our own modified versions of some of the CodeMirror code :\

Happy to update CodeMirror as a separate patch